web/db/migrations/20260902060000_remove_e2b_daytona_vm_providers/migration.sql

```
@@ -0,0 +1,35 @@
+-- at the end is the safety interlock: it fails if anything still references the
+-- old type, so a missed column aborts the whole transaction instead of silently
+-- leaving a split schema.
+--
+-- Rows still naming a removed provider are rewritten to 'freestyle'
+-- first. Those machines are unreachable regardless (both drivers are gone);
+-- this keeps history and usage rows readable rather than dropping them, at the
+-- cost of mislabelling their provider. Destroy any live machines on the
+-- removed providers BEFORE applying this migration — afterwards nothing in the control plane can
+-- address them.
+
+UPDATE "cloud_vms" SET "provider" = 'freestyle' WHERE "provider" IN ('e2b', 'daytona');
```

SQLFluff (4.3.0)

[error] The 'WHERE' keyword should always start a new line.

(LT14)